### DISCUSSION TITLE
onChange does not trigger when you highlight and delete an input

### DISCUSSION THREAD

**Discussion by Cants820** [react-hook-form/react-hook-form#8466] [Q&A]
Hello,

I am having a difficult time trigger a onChange event " you highlight and delete a textbox". I was wondering if react-hook-forms was a solution for this since i am using uncontrolled inputs. 
Here is a sample of one of my inputs which onChange does not trigger: 
`<input type="text" name="imdgClass" maxLength='10' onChange={handleChange} defaultValue={imdgClass}  label="Class" register={register} disabled={viewOnly} />`

My proposed solutions to the issue:
 -  use a controller component instead 
 - add value attribute alongside with defaultValue (which is bad practice)

**Comment by bluebill1049** [comment]
share a codesandbox?
